RELEASING.md

```diff
@@ -77,7 +82,7 @@ and assign it to @bencochran or @aquageek.*
    ```bash
    git checkout master
    git pull
-   git merge --no-ff v0.1-fixes
+   git merge --no-ff release-v0.0.x
```

Comment:
Your previous examples used the release branch `release-v0.1.x`, you probably want to use the same here.
```suggestion
   git merge --no-ff release-v0.1.x
```

Reply:
I had used `release-v0.0.x` for the "fix-branch". But maybe that's a very subtle difference :D

Reply:
Ah, I missed that